includes/class-omise-payment-methods.php

```diff
@@ -0,0 +1,41 @@
+<?php
```

Comment:
Does this need to be a singleton? It has no state, it is fully self contained. A very simple class with a public static array of the methods would be much simpler. This is overcomplicating things

Reply:
yeah, no particular reason to be singleton. Let me think if this will be in some other uses. If no then will go with what you suggested.